assets/css/main.css

```diff
@@ -151,3 +151,41 @@ textarea {
 .relative-position {
   position: relative;
 }
+ .success {
+  position:absolute;
+  height: 55px;
+  background-color: #0BC475;
+  top:90px;
+  left:0px;
+  right:0px;
+  border-radius: 5px;
+}
+.error {
+  position:absolute;
+  height: 55px;
+  background-color: #E85F5F;
+  top:90px;
+  left:0px;
+  right:0px;
+  border-radius: 5px;
+}
```

Comment:
`.success` and `.error` are sharing many classes. Can we create a `.swal-basic` class and reuse properties?

Reply:
Resolved

Reply:
Swal className property is not taking to classNames fro somee reason such as 'swal-basic success' is throwing DOM error as there can not be spaces between class names and ['swal-basic', 'success'] is not applying any style to the element